actions/project.go

```diff
@@ -70,6 +71,36 @@ func DownloadTemplate(c *cli.Context) {
 	}
 }
 
+// testIsExtension tests if a project is an extension project and run associated commands as necessary
+func testIsExtension(projectPath string) (string, error) {
+
+	extensions, err := apiroutes.GetExtensions()
+	if err != nil {
+		return "", err
```

Comment:
As the func `GetExtensions()` can return an error, maybe it would be good to display it here at the top level of the func call in this if block. To do this you could use `log.printf()` or `log.fatal()`. The reason i suggest using the log library in this case is because it prints to stderr by default.